README.md

```diff
@@ -147,6 +147,14 @@ You can find, in all model project directories, a ``README.md`` file that descri
 
 ### Launch the Training
 
+The trainings are launched using the Lightning CLI, configured using YAML files. In the configuration files, you can use the variables interpolation feature from [Omegaconf](https://omegaconf.readthedocs.io/en/latest/), to set some variables.
+
+By default, you can set 2 environment variables that will be passed to the Lightning CLI configuration files:
+- **DATADIR**: the path where your dataset is stored,
+- **LOGDIR**: the path where the lightning logs, artifacts and plots will be stored.
+
```

Comment:
There is also the environment variable `SLURM_GPUS_ON_NODE` that has to be set. Should we use a generic name in the case we do not use Slurm ?

Reply:
I didn't document this variable because, this is Slurm which set it (if Slurm is used), otherwise the default value applies. 

I guess, in general, the user will update the config file to set the exact values on the trainer part, for example theexact number of device he want to use. This is more an example of what can be done to configure the training using Slurm env, but it is non-blocking for the user who try to train without any config change, because of the default value.